server/lib/friends.js

```diff
@@ -222,6 +223,55 @@ function quitFriends (callback) {
   })
 }
 
+function removeFriend (podId, callback) {
+  // Stop pool requests
+  requestScheduler.deactivate()
+
+  waterfall([
+    function flushRequests (callbackAsync) {
+      requestScheduler.flush(err => callbackAsync(err))
```

Comment:
You don't need this. If the pod doesn't exist in our database anymore we won't do requests on it.